dev_guide/routes.adoc

```diff
@@ -133,6 +133,22 @@ TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
 
+[[creating-labeled-routes]]
+== Creating Routes with Labels
+
+You can create a route based on labels. A explanation how can be found at
+link:../architecture/core_concepts/routes.adoc#router-sharding
+
+How router sharding is created is explained at
+link:../install_config/install/deploy_router.adoc#creating-router-shards[Creating Router Shards]
+you need to add the labels for the dedicated routes.
+
```

Comment:
Similarly to the first sentence, how about if we use:

```asciidoc
The labels can be
link:../install_config/install/deploy_router.html#creating-router-shards[selected by routers]
to create router shards.
```

and remove the intervening blank line, so that the two sentences form one paragraph?